tests/modules/nf-core/ashlar/zero_uuid.nf

```diff
@@ -0,0 +1,32 @@
+nextflow.enable.dsl=2
+
+process ZERO_UUID {
+
+    input:
+    each path(file_in)
```

Comment:
I propose this takes a file path _and_ the numeric offset of the UUID, which we will pre-calculate for each test output file and hard-code in the test script calls to ZERO_UUID. That simplifies this code dramatically. We could even write this purely in Groovy since it's just some simple file operations, but that optimization can be done some other time.

Reply:
Ok, sure, that makes sense.  sounds good to me.

Reply:
I just pushed a simplified version of zero_uuid.nf that gets the input file path in groovy so I don't have to parse it out in bash.